source/Nevermore.IntegrationTests/CustomTypes/CustomSingleTypeInColumnFixture.cs

```diff
@@ -0,0 +1,114 @@
+using System.Collections.Generic;
+using FluentAssertions;
+using Nevermore.Contracts;
+using Nevermore.Mapping;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace Nevermore.IntegrationTests.CustomTypes
+{
+    public class CustomSingleTypeInColumnFixture : FixtureWithRelationalStore
+    {
+        [Test]
+        public void ShouldWorkInColumn()
+        {
+            using (var transaction = Store.BeginTransaction())
+            {
+                var model = new ModelTypeWithColumn
+                {
+                    CustomTypeUsedAsAProperty =  new CustomTypeUsedAsAProperty
+                    {
+                        Name = "foo",
+                        Value = 10
+                    }
+                };
+                transaction.Insert(model);
+
+                var read = transaction.Query<CustomTypeWithColumnToTestSerialization>()
```

Comment:
I still find this test a bit confusing - after a few minutes I think I figured out that it's testing that it can read and write it as both a string and a custom type, but I'm not sure. If the example was more real world or had comments maybe that might help?